package.json

```diff
@@ -41,6 +41,7 @@
     "@types/auth0-js": "^8.11.11",
     "@types/graphql": "^14.0.6",
     "@types/ws": "^6.0.1",
+    "angular": "^1.7.7",
```

Comment:
Do not add this, this will import AngularJS. You don't need to modify package.json